test/mocha/050-client-api.js

```diff
@@ -3,20 +3,42 @@
  */
 'use strict';
 
-const {_client} = require('bedrock-ledger-consensus-continuity');
+const {_client, _localPeers} = require('bedrock-ledger-consensus-continuity');
 const {config, util: {BedrockError}} = require('bedrock');
 
 describe('Client API', () => {
   describe('notifyPeer', () => {
+    // NOTE: this name is really bad. peerId is actually a database record
+    // with tons of information including private key material
```

Comment:
`peerId` is a database record and not a peer ID? That doesn't sound right. Can you look at this?

Reply:
Looked into it and made an issue: https://github.com/digitalbazaar/bedrock-ledger-consensus-continuity/issues/250

Reply:
Let's fix that issue here now (seems like a one line fix) and remove this note.

Reply:
addressed here: https://github.com/digitalbazaar/bedrock-ledger-consensus-continuity/pull/249/commits/ac9aec6e68f0b3681c6b1508bd2ba7e40150f39e and the commit after it.